WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Status/JetpackBackupStatusCoordinator.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+protocol JetpackBackupStatusView {
+    func render(_ backup: JetpackBackup)
+    func showError()
+    func showComplete()
+}
+
+class JetpackBackupStatusCoordinator {
+
+    private let service: JetpackBackupService
+    private let site: JetpackSiteRef
+    private let view: JetpackBackupStatusView
+
+    init(site: JetpackSiteRef,
+         view: JetpackBackupStatusView,
+         service: JetpackBackupService? = nil,
+         context: NSManagedObjectContext = ContextManager.sharedInstance().mainContext) {
+        self.service = service ?? JetpackBackupService(managedObjectContext: context)
+        self.site = site
+        self.view = view
+    }
+
+    func start() {
+        service.prepareBackup(for: site, success: { [weak self] backup in
+            self?.view.render(backup)
+            self?.pollBackupStatus(for: backup.downloadID)
+        }, failure: { [weak self] error in
+            DDLogError("Error preparing downloadable backup object: \(error.localizedDescription)")
+
+            self?.view.showError()
+        })
+    }
+
+    private func pollBackupStatus(for downloadID: Int) {
+        Timer.scheduledTimer(withTimeInterval: 1, repeats: true) { [weak self] timer in
```

Comment:
@zwarm 👋
On Android, what polling interval are you using for fetching the backup status?
I'm polling the backup status every second (just for now)